src/core/coap/coap_secure.cpp

```diff
@@ -320,6 +320,11 @@ CoapSecure &CoapSecure::GetOwner(const Context &aContext)
     return coap;
 }
 
+void CoapSecure::HandleRetransmissionTimer(Timer &aTimer)
+{
+    GetOwner(aTimer).Coap::HandleRetransmissionTimer();
```

Comment:
I am not sure if I understand this. We seems to be calling the base class method here, is this correct?